test-requirements.txt

```diff
@@ -1 +1 @@
-Django==1.11.5
\ No newline at end of file
+Django==1.11.5
```

Comment:
?

Reply:
oh yeah, my sublime is set up to automatically add a new line to the end of files whenever I hit save.  That's the only difference here.